Library/Homebrew/dev-cmd/bump.rb

```diff
@@ -288,7 +288,12 @@ def livecheck_result(formula_or_cask)
   }
   def retrieve_pull_requests(formula_or_cask, name, state:, version: nil)
     tap_remote_repo = formula_or_cask.tap&.remote_repo || formula_or_cask.tap&.full_name
-    pull_requests = GitHub.fetch_pull_requests(name, tap_remote_repo, state: state, version: version)
+    begin
+      pull_requests = GitHub.fetch_pull_requests(name, tap_remote_repo, state: state, version: version)
+    rescue GitHub::API::ValidationFailedError => e
+      odebug "Error fetching pull requests for #{formula_or_cask} #{name}: #{e}"
+    end
+
```

Comment:
```suggestion
    pull_requests = begin
      GitHub.fetch_pull_requests(name, tap_remote_repo, state: state, version: version)
    rescue GitHub::API::ValidationFailedError => e
      odebug "Error fetching pull requests for #{formula_or_cask} #{name}: #{e}"
      nil
    end

```

feels a little more explicit